src/test/java/redis/clients/jedis/JedisPoolTest.java

```diff
@@ -22,13 +22,15 @@
 
 public class JedisPoolTest {
 
-  private static final HostAndPort hnp = HostAndPorts.getRedisServers().get(0);
+  private static final EndpointConfig endpoint1 = HostAndPorts.getRedisEndpoint("standalone0");
```

Comment:
I would name the variables to match the lookup key, e.g. `endpointStandalone0`, or anything similar. At first look it is a bit confusing that the key are numbered from 0, the variables from 1.